pkg/router/routes.go

```diff
@@ -16,7 +16,7 @@ import (
 
 // NewRouter takes in handler Dependencies and returns mux.Router with default routes
 // and if debug mode is enabled then it also adds pprof routes.
-// It also, adds a PathPrefix to the catch all router if config.Source().PathPrefix is set
+// It also, adds a PathPrefix to catch all of the router if config.DataSource().PathPrefix is set
```

Comment:
I think earlier statement was correct. You might already know this but just to add more clarity here, a `catch all` route is basically a route with `*` wildcard or `.+` regex which takes any path and forwards that to the handler.
`catch all of the router` doesn't really make sense to me. I do agree that router should be changed to route though or the statement can be `catch all route of the router`. WDYT?

Reply:
Sure, will update the comment back, thanks for clarifying